src/api.js

```diff
@@ -1,11 +1,72 @@
+//GET requests
 export const sleepDataAPI = fetch("http://localhost:3001/api/v1/sleep")
-                              .then(response => response.json())
+  .then(response => response.json())
 
 export const userDataAPI = fetch("http://localhost:3001/api/v1/users")
-                            .then(response => response.json())
+  .then(response => response.json())
 
 export const activityDataAPI = fetch("http://localhost:3001/api/v1/activity")
-                                .then(response => response.json())
+  .then(response => response.json())
 
 export const hydrationDataAPI = fetch("http://localhost:3001/api/v1/hydration")
-                                  .then(response => response.json())
+  .then(response => response.json())
+
+
+//POST requests
+export const sleepDataPost = (dataFormEntry) => {
+  fetch("http://localhost:3001/api/v1/sleep", {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(dataFormEntry)
+    })
+    .then(response => response.json())
+    .then(json => console.log(json))
+    .catch(err => console.log('too much sauce'))
```

Comment:
Lookin' good! We should add `.catch()` to the GET request as well. Did you say you were working on making these dynamic?